### PR TITLE
iOS-3435 Chat Input | Support mention without icon (to chat)

### PR DESCRIPTION
**Pull request by mgolovko**
https://github.com/user-attachments/assets/1b47ebc0-a855-4b96-aeed-1b2c50137575

### CODE REVIEW

File: Anytype/Sources/PresentationLayer/Common/UIKit/AnytypeUITextView/AnytypeUITextView.swift [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        let newPosition = super.position(from: position, in: direction, offset: offset)
+        
+        guard let newPosition else { return nil }
+
+        return validatePosition(newPosition: newPosition, direction: direction == .right ? .right : .left)
+    }
+    
+    // MARK: - Private
+    
+    enum ValidateDirection {
```
Comment (joe-pusya) [inline Anytype/Sources/PresentationLayer/Common/UIKit/AnytypeUITextView/AnytypeUITextView.swift:62]: private?

File: Anytype/Sources/PresentationLayer/Common/UIKit/AnytypeUITextView/AnytypeUITextView.swift [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        case `default`
+        case left
+        case right
+    }
+    
+    private func validatePosition(newPosition: UITextPosition, direction: ValidateDirection) -> UITextPosition? {
+        
+        let beingNew = self.offset(from: beginningOfDocument, to: newPosition)
```
Comment (joe-pusya) [inline Anytype/Sources/PresentationLayer/Common/UIKit/AnytypeUITextView/AnytypeUITextView.swift:70]: beginningNew?

**Review by joe-pusya — APPROVED** [review]

**Comment by joe-pusya** [comment]
🤯

File: Anytype/Sources/PresentationLayer/Common/UIKit/AnytypeUITextView/AnytypeUITextView.swift (hunk 2 above)
Comment (mgolovko) [reply Anytype/Sources/PresentationLayer/Common/UIKit/AnytypeUITextView/AnytypeUITextView.swift:70]: Fixed

File: Anytype/Sources/PresentationLayer/Common/UIKit/AnytypeUITextView/AnytypeUITextView.swift (hunk 1 above)
Comment (mgolovko) [reply Anytype/Sources/PresentationLayer/Common/UIKit/AnytypeUITextView/AnytypeUITextView.swift:62]: Fixed

**Pull request merged**